aries-site/src/examples/templates/forms/SimpleSignUpExample.js

```diff
@@ -191,6 +204,7 @@ export const SimpleSignUpExample = () => {
               }
             >
               <TextInput
+                style={{ backgroundColor: backgroundError }}
```

Comment:
Soft opinion, but would we want to wrap this textinput in a box and set the background on there as a prop? using the `style` prop feels a little anti grommety

Reply:
yeah this will also change after talking with Greg we dont want validation on blur so the whole background will be taken out